client/src/app/components/task-list/task-list/task-list.component.html

```diff
@@ -3,3 +3,44 @@
     Add Task
   </button>
 </div>
+
+<div class="container" style="padding: 10px">
+  <div class="row">
+    <div *ngFor="let task of tasks" class="col-sm-3">
+      <div
+        class="card mb-3"
+        [ngClass]="{
+          'bg-light': task.status == 'To Do',
```

Comment:
@networkflow I store statuses in `db` , most likely it is not going to change. Is it ok if I just do enum.
e.g `enum TaskStatus {
toDo: "To Do",
inProgress: "In Progress",
stuck: "Stuck",
done: "Done",
}`

Or should I get this from db?

Reply:
I'm not completely sure if I understand your question: are you asking about creating a new `enum TaskStatus` in the `client` app to match `TaskStatus` in server? I'm not sure what "get this from the DB" would mean: I'm not sure how you could get all the possible status values in a nice way from the DB.

Personally I would treat `TaskStatus` the same way you treat the `Task` DTO objects: either put them in a shared folder, or have separate `TaskStatus` objects on the client/server and have big comments in the code in both files telling you to keep the 2 files in sync with each other.